### PR TITLE
chore: cleanup anything related to public squads submission

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Delete anything related to public squads submission

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

MI-534

### Preview domain
https://mi-534.preview.app.daily.dev

### CODE REVIEW

**Review by nensidosari — APPROVED** [review]

File: packages/shared/src/components/checklist/actionUtils.ts [hunk 1]
```diff
@@ -12,10 +12,3 @@ export const getEditActions = (squad: Squad): Action[] => {
     ? [{ type: ActionType.EditSquad, completedAt: new Date() }]
     : [];
 };
-
-export const getPublicActions = (squad: Squad): Action[] => {
```
Comment (rebelchris) [inline packages/shared/src/components/checklist/actionUtils.ts:16]: this one is already removed in this branch?
https://github.com/dailydotdev/apps/blob/8ffbe0687a507702747ad9e26efeec642a7e33a9/packages/shared/src/hooks/useSquadChecklist.tsx#L146

**Review by rebelchris — APPROVED** [review]
Just one sanity check, but probably fine

File: packages/shared/src/components/checklist/actionUtils.ts (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/checklist/actionUtils.ts:16]: Right. Been looking for this. Thank you! 🚀

Comment (sshanzel) [reply packages/shared/src/components/checklist/actionUtils.ts:16]: Ah, it has been deleted from this file already. Thought the link was towards this PR. We should be good.

**Pull request merged**